core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1674,7 +1674,11 @@ private[spark] object Utils extends Logging {
 
   /**
    * NaN-safe version of `java.lang.Double.compare()` which allows NaN values to be compared
-   * according to semantics where NaN == NaN and NaN &gt; any non-NaN double.
+   * according to semantics where
+   *
+   * {{{
+   * NaN == NaN and NaN > any non-NaN double
```

Comment:
Just change this to "is greater than". It's not really code we're describing here but semantics of numeric ordering